code/modules/wiremod/shell/brain_computer_interface.dm

```diff
@@ -290,21 +290,15 @@
 	occupant_typecache = typecacheof(/mob/living/carbon)
 
 /obj/machinery/bci_implanter/on_deconstruction()
-	var/obj/item/organ/internal/cyberimp/bci/bci_to_implant_resolved = bci_to_implant?.resolve()
-	bci_to_implant_resolved?.forceMove(drop_location())
-	bci_to_implant = null
+	drop_stored_bci()
 
 /obj/machinery/bci_implanter/Destroy()
-	QDEL_NULL(bci_to_implant)
+	drop_stored_bci()
 	return ..()
 
 /obj/machinery/bci_implanter/examine(mob/user)
-	. = ..()
-
-	if (isnull(bci_to_implant?.resolve()))
-		. += span_notice("There is no BCI inserted.")
-	else
-		. += span_notice("Right-click to remove current BCI.")
+	. += span_notice(bci_to_implant ? "Right-click to remove current BCI." : "There is no BCI inserted.")
+	return ..()
```

Comment:
This doesn't work and will not include the notice you added. Revert this change and replace `if (isnull(bci_to_implant?.resolve()))` with `if (isnull(bci_to_implant))`.

Reply:
Will do.